scripts/add_electricity.py

```diff
@@ -338,6 +348,24 @@ def attach_wind_and_solar(
                 p_max_pu=ds["profile"].transpose("time", "bus").to_pandas(),
             )
 
+        if suptech == "offwind":
+            continue
+            # df.carrier.mask(df.technology=="Offshore", "offwind-ac",inplace=True)
+
+        df.carrier.mask(df.technology == "Onshore", "onwind", inplace=True)
```

Comment:
@yerbol-akhmetov  Looking soon over the PR. Just a little Q. Is this line just necessary because the custom_powerplant data is not stored correctly / or the docs are described such giving false instructions on storing the data?

Reply:
@pz-max, I have added this lines because by default `onwind` is stored as `wind` based on PPM standards. Please, see the figure below: 
![image](https://github.com/pypsa-meets-earth/pypsa-earth/assets/113768325/c1803511-8a54-4a4e-b3e7-297714c69e81)